src-v2/Not/Not/Extensions/DateTimeOffsetExtension.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Not.Extensions;
+public static class DateTimeOffsetExtension
+{
+    public static DateTimeOffset ToDateTimeOffset(this DateTime time)
+    {
+        DateTime timeWithSpecifiedKind = DateTime.SpecifyKind(time, DateTimeKind.Local);
+        DateTimeOffset offsetTime = timeWithSpecifiedKind;
+        return offsetTime;
+    }
+    public static DateTimeOffset? ToDateTimeOffset(this TimeSpan? timeToBeAdded)
```

Comment:
I don't think we should involve nullability in extensions, unless we're extending `Nullable` itself. Instead you should make sure the method is not executed on null value. See [null-conditional operator](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/operators/member-access-operators#null-conditional-operators--and-)